modules/javafx.graphics/src/test/java/test/javafx/concurrent/ScheduledServiceTest.java

```diff
@@ -91,8 +92,8 @@ public class ScheduledServiceTest extends ServiceTestBase {
         };
     }
 
-    @Override public void setup() {
-        super.setup();
+    @Override public void setup(TestServiceFactory factory) {
```

Comment:
could have inserted a newline after the annotation...

Reply:
this particular test is not parameterized, so you could use `@BeforeEach` to setup the factory.  Same for the `ServiceLifecycleTest`.

it won't work for `ServiceExceptionTest` because it is parameterized.

but this code is ok too.

Reply:
Changed it to use `@BeforeEach` in non-parametrized classes